app/views/support_interface/tasks/confirm_delete_test_applications.html.erb

```diff
@@ -0,0 +1,26 @@
+<% content_for :before_content, govuk_back_link_to(support_interface_tasks_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: nil,
+      url: support_interface_run_task_path('delete_test_applications') do |f|
+    %>
+      <h1 class="govuk-heading-xl">
+        Delete test applications
+      </h1>
+
+      <p class="govuk-body">
+        This will delete all candidates with emails that end in "@example.com", their applications and associated data. Are you sure?
```

Comment:
```suggestion
        This will delete all candidates with emails that end in “@example.com”, their applications and associated data. Are you sure?
```